src/commands/sync/compare.ts

```diff
@@ -39,19 +39,21 @@ async function compareHandler(this: Command, pattern: string) {
 
   console.log('Your code project and Tolgee project are out of sync.');
   if (diff.added.length) {
-    console.log(ansi.green.bold(`${diff.added.length} new strings`));
+    const key = diff.added.length === 1 ? 'key' : 'keys'
+    console.log(ansi.green.bold(`${diff.added.length} new ${key} found in your code project`));
     for (const key of diff.added) {
-      printKey(key, 'added');
+      printKey(key, false);
     }
 
     // Line break
     console.log('');
   }
 
   if (diff.removed.length) {
-    console.log(ansi.red.bold(`${diff.removed.length} removed strings`));
+    const key = diff.added.length === 1 ? 'key' : 'keys'
+    console.log(ansi.red.bold(`${diff.removed.length} ${key} exist on Tolgee but were not found in your code project`));
```

Comment:
Same here, it could be "10 unused keys"

Reply:
For this one I'm unsure, unused may be weird to inexperienced users and having the CLI clearly tell things might help. But maybe I'm making it too foolproof here 😅 

I'll go back to the simple way here too I guess.